client/src/pages/Dashboard.js

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from "react";
+
+import { Grid } from "@material-ui/core";
+import { withStyles } from "@material-ui/core/styles";
+
+import Mentions from "./Mentions";
+import Platforms from "./Platforms";
+import Navbar from "./Navbar";
+
+const dashboardStyle = theme => ({
+  rightGridContainer: {
+    borderLeft: '1px solid black',
+    height: '100%'
+  }
+});
+
+class Dashboard extends Component {
+  state = {
+    platforms: [
+      {name: "Reddit", inUse: true},
+      {name: "Twitter", inUse: true},
+      {name: "Facebook", inUse: true},
+      {name: "Amazon", inUse: true},
+      {name: "Forbes", inUse: true},
+      {name: "Shopify", inUse: true},
+      {name: "Business Insider", inUse: true}
+    ],
+    mentions: [
+      {title: 'thing happened', platform: 'Reddit', desc: 'qwerty'},
+      {title: 'consume', platform: 'Forbes', desc: '12345'},
+      {title: 'buy', platform: 'Shopify', desc: 'abc123'},
+      {title: 'do', platform: 'Business Insider', desc: 'aedsfadwsf'}
+    ],
```

Comment:
Assuming these are just placeholders for now, but will probably need to include other data about each mention to support other sorting capabilities.